Sources/SourceKitLSP/Swift/VariableTypeInfo.swift

```diff
@@ -34,10 +66,12 @@ struct VariableTypeInfo {
           let hasExplicitType: Bool = dict[keys.variable_type_explicit] else {
       return nil
     }
+    let tokenAtOffset = snapshot.tokens.syntaxTree?.token(at: AbsolutePosition(utf8Offset: offset))
 
     self.range = startIndex..<endIndex
     self.printedType = printedType
     self.hasExplicitType = hasExplicitType
+    self.canBeFollowedByTypeAnnotation = tokenAtOffset?.canBeFollowedByTypeAnnotation ?? true
```

Comment:
Would a `false` default be better here?

Reply:
Not really. `canBeFollowedByTypeAnnotation` is a blacklist-based method that also defaults to `true`. That’s why I think we should also default to `true` here for consistency.